source/js/localization.js.es6

```diff
@@ -164,8 +164,12 @@ const localization = {
   },
 }
 
-;(() => {
-  document.addEventListener('DOMContentLoaded', () => {
-    localization.onLoad()
-  })
-})()
+;(function() {
+  if (document.readyState === 'loading') {
+    document.addEventListener('DOMContentLoaded', () => {
+      localization.onLoad();
+    });
+  } else {
+    localization.onLoad();
+  }
+})();
```

Comment:
I'm not sure why you need the `document.readyState === 'loading'` check before adding the `DOMContentLoaded` listener?

Reply:
Ah, this is now `<script src="…" defer async >` so the script _might_ load after the content has loaded and is non-blocking.

Technically…maybe a bad change with the `defer async`!  The user will see navigation they cannot interact with in that scenario.

Definitely not something I know a lot of actually..

Reply:
Ah right the `<scripts>` are in the `<head>` 😞 . I think, you could have just `defer` as it's non blocking and just keep the `DOMContentLoaded` as it was:

**defer**
> `DOMContentLoaded` event handler waits for the deferred script. It only triggers when the script is downloaded and executed.

**async**

> `DOMContentLoaded` may happen both before and after async, no guarantees here.

🤯

Reply:
Yeah…I'll let you have a think as I kind of reached my limit here if QA seems to have issues with this.

Maybe it should just be blocking, but ideally scripts aren't—I'm looking at Chrome Lighthouse a lot with this too.

Reply:
I agree - scripts shouldn't be blocking. Can you see this ever getting into the `else` of the `if (document.readyState === 'loading') {` block?

Reply:
I was worried about double-initializing 😟  I'd get two event handlers?